Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -80,6 +80,13 @@ type BreadcrumbItem = {
   onClick?: () => void;
 };
 
+// field types
+const Types = {
+  Dialog: 'dialog',
+  Trigger: 'trigger',
+  Action: 'action',
+};
+
```

Comment:
See below comment - I'm wondering about changing this slightly so we don't have to worry about that "unsafe regex" linter rule.